test/test_belief_propagation.jl

```diff
@@ -133,42 +70,8 @@ using Test: @test, @testset
   rdm = array((rdm * combiner(inds(rdm; plev=0)...)) * combiner(inds(rdm; plev=1)...))
   rdm /= tr(rdm)
 
-  eigs = eigvals(rdm)
+  eigs = Float64.(eigvals(rdm))
```

Comment:
What's this conversion for?

Reply:
Was having some failing tests on Windows because of eigs that were `~-1E-17` so I refactored the test and this now essentially checks that all values in `eigs` are real up to machine precision.
But I should change this back to check the more explicit way.

Reply:
Have changed that now